CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ All notable changes to the Pony compiler and standard library will be documented
 ### Changed
 
 - Remove unnecessary argument to `Map.sub` ([PR #3275](https://github.com/ponylang/ponyc/pull/3275))
+- Rename MaybePointer to NullablePointer
```

Comment:
Can you update this to include a link to the PR you opened and rebase against master so this would be mergeable?